.github/workflows/documentation.yaml

```diff
@@ -7,6 +7,10 @@ on:
   # Enables a manual trigger.
   workflow_dispatch:
 
+concurrency:
+  group: docs-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: false
```

Comment:
Why not abort documentation in progress?

Reply:
Because it uses rsync to copy the documentation to the server and I would like it not to interrupt that